crates/matrix-sdk-base/src/client.rs

```diff
@@ -93,7 +94,7 @@ pub struct BaseClient {
     #[cfg(feature = "e2e-encryption")]
     olm_machine: Arc<RwLock<Option<OlmMachine>>>,
     /// Observable of when a user is ignored/unignored.
-    pub(crate) ignore_user_list_changes: SharedObservable<()>,
+    pub(crate) ignore_user_list_changes: SharedObservable<Vec<String>>,
```

Comment:
Added the list of user IDs in here to keep in line with the other publishers we have.